test/org/sagebionetworks/bridge/models/RequestInfoTest.java

```diff
@@ -0,0 +1,81 @@
+package org.sagebionetworks.bridge.models;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.Test;
+
+import org.sagebionetworks.bridge.TestConstants;
+import org.sagebionetworks.bridge.TestUtils;
+import org.sagebionetworks.bridge.json.BridgeObjectMapper;
+import org.sagebionetworks.bridge.json.DateUtils;
+import org.sagebionetworks.bridge.models.studies.StudyIdentifier;
+import org.sagebionetworks.bridge.models.studies.StudyIdentifierImpl;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.collect.Sets;
+
+import nl.jqno.equalsverifier.EqualsVerifier;
+
+public class RequestInfoTest {
+
+    private static final StudyIdentifier STUDY_ID = new StudyIdentifierImpl("test-study");
+    private static final String USER_ID = "userId";
+    private static final ClientInfo CLIENT_INFO = ClientInfo.parseUserAgentString("app/20");
+    private static final String USER_AGENT_STRING = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.116 Safari/537.36";
+    private static final LinkedHashSet<String> LANGUAGES = TestUtils.newLinkedHashSet("en", "fr");
+    private static final DateTimeZone PST = DateTimeZone.forOffsetHours(-7);
+    private static final DateTime ACTIVITIES_REQUESTED_ON = DateUtils.getCurrentDateTime();
+    private static final DateTime SIGNED_IN_ON = ACTIVITIES_REQUESTED_ON.minusHours(4);
```

Comment:
I'll create some tests for these. This use of time zones is informational only (I'm assuming the researcher would be interested in the time in the user's local time, though that may not be true and we might eventually enforce UTC instead). 

In JavaScript, the tools for manipulating dates are very poor and it isn't easy to show a timestamp in an alternate timezone, even if I have it. If we don't have it, it's okay to show the times in UTC and I can enforce this in case the server is in yet another time zone.